quotefault/templates/bootstrap/hidden.html

```diff
@@ -0,0 +1,100 @@
+{% extends "bootstrap/base.html" %}
+
+{% block styles %}
+    <link rel="stylesheet" href="/static/css/votes/upvote.css">
+{% endblock %}
+
+{% block body %}
+    <script type="text/javascript">
+        function unhide(id){
```

Comment:
:cry: consistent indentation?

Reply:
I could be blind but I don't see what's inconsistent here

Reply:
the `function` declaraction and `var modal = ...` are at the same level... that's bad!